src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,47 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    public static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
+        Movie theLostCity = new Movie("The Lost City");
+        theLostCity.setDescription("A reclusive romance novelist on a book tour with her...");
+        movieService.add(theLostCity);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        final CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        CinemaHall cinemaHall1 = new CinemaHall();
```

Comment:
```suggestion
        CinemaHall smallCinemaHall = new CinemaHall();
```

it's better not to use numbers in variable names